DBSCE/Overview.md

```diff
@@ -0,0 +1,275 @@
+
+## Authors
+
+- [Sameera Gajjarapu](sameera.gajjarapu@microsoft.com), Microsoft
+- [Aleksander Tokarev](alextok@microsoft.com), Microsoft
+
+## Contributors
+
+- [Olga Dalton](), Microsoft
+- [Kristian Monsen](), Google
+- [Phil Leblanc](), Google
+- [Sebastian](), Google
+- [Arnar Birgisson](), Google
+- [Pamela Dingle](), Microsoft
+- [Paul Garner](), Microsoft
+- [Erik Anderson](), Microsoft
+- [Will Bartlett](), Microsoft
+- [Kai Song](), Microsoft
+- [Amit Gusain](), Microsoft
+
+## Participate (TBD links)
+
+- [Issue tracker]()
+- [Discussion forum]
+
+## Overview
+
+Device Bound Session Credentials for Enterprise - DBSC(E), is an enhancement to the existing [DBSC](https://github.com/wicg/dbsc) proposal. It refines the key generation mechanism resulting in additional security for enterprise use cases. It aims to provide a mechanism for enterprise and advanced browser customers to be able to deploy enhanced/customised device binding for any browser session, hence protecting against session hijacking and cookie theft.
+
+## Why DBSC(E)?
+
+While the original DBSC proposal enables browsers to bind session cookies to a device, it still remains vulnerable to "on device" malware. Such a malware, if present on the device, can inject its own binding keys when the DBSC session is established during any signin operaton. If a DBSC session is already established when the malware gains access to the system, the malware can force a new signin operation, and potentially hijack all subsequent sessions. Any upcoming sessions after this, even with DBSC, will not be reliable. Hence a temporary malware presence in the system can result in permanent session compromise in certain cases.
+
+DBSC(E) aims to mitigate this risk by introducing the concept of once-in-a-lifetime protected [device registration](#device-registration) operation and binds all the future sessions to binding keys that can be cryptographically proven to be on the same device. DBSC(E) is able to provide this risk mitigation if the device registration is performed when there is no malware on the device (a state referred to as ["clean room"](#device-registration-client)) e.g. an organization registering a device before giving a device to an employee. Device registration is also expected to be a once-in-a-lifetime protected operation, hence the user will not be required to perform this operation again, reducing opportunities for malware to compromise a user session. 
+
+Therefore, if a device registration is executed in a clean room and precedes any sign in sessions, DBSC(E) makes it impossible for a malware to bind session cookies to malicious binding keys during a sign in operation that implements DBSC(E). However, it is to be noted that, DBSC(E) doesn't protect a given session if the malware is present during the device registration or if the malware is persistent on the device and uses device-bound sessions to exfiltrate application data rather than the sessions.
```

Comment:
Discussed with Paul and @sameerag to reword this slightly, adding a note

Reply:
Addressed offline.